src/sidebar/directive/annotation.js

```diff
@@ -291,8 +326,7 @@ function AnnotationController(
       // example there's no vm.annotation.highlight: true.  Instead a highlight is
       // defined as an annotation that isn't a page note or a reply and that
       // has no text or tags.
-      var isPageNote = (vm.annotation.target || []).length === 0;
-      return (!isPageNote && !isReply(vm.annotation) && !vm.hasContent());
+      return (!isPageNote(vm.annotation) && !isReply(vm.annotation) && !vm.hasContent());
```

Comment:
The `isPageNote` function already checks `!isReply`, so you can remove the `&& !isReply(vm.annotation)` clause here.